packages/react-testing/README.md

```diff
@@ -375,7 +375,7 @@ export const mountWithGraphQL = createMount<Options, Context, true>({
   async afterMount(root, {skipInitialGraphQL}) {
     // This is a temporary hack to make GraphQL resolution behave pseudo-synchronously
     // to avoid warnings about setting state outside of act() blocks.
-    root.graphQL.on('pre-resolve', () => {
+    root.context.graphQL.on('pre-resolve', () => {
```

Comment:
fix example

Reply:
what is this fixing?

Reply:
that `graphQL` is actually part of the `context` object, not `root`
the example won't work basically.